digits/standard-networks/tensorflow/alexnet_slim.py

```diff
@@ -0,0 +1,38 @@
+from model import Tower
```

Comment:
I think Tim wants to have both version to show that if someone want to do slim, they can... @TimZaman  can you confirm?